pkg/pillar/scripts/device-steps.sh

```diff
@@ -7,21 +7,24 @@ WATCHDOG_PID=/run/watchdog/pid
 WATCHDOG_FILE=/run/watchdog/file
 CONFIGDIR=/config
 PERSISTDIR=/persist
+PERSIST_STATUS_DIR=/persist/status
 PERSIST_CERTS=$PERSISTDIR/certs
 BINDIR=/opt/zededa/bin
 TMPDIR=/persist/tmp
 ZTMPDIR=/var/tmp/zededa
 DPCDIR=$ZTMPDIR/DevicePortConfig
 FIRSTBOOTFILE=$ZTMPDIR/first-boot
-GCDIR=$PERSISTDIR/config/GlobalConfig
+GCDIR=$PERSISTDIR/config/ConfigItemValueMap
+LAST_IMAGE_VERSION_FILE=$PERSIST_STATUS_DIR/last-image-version
```

Comment:
I don't think the name of this is descriptive enough @kalyan-nidumolu 

Also, there maybe a bigger issue of whether we're tracking EVE versions of versions of the /persist "schema"

Reply:
Right now.. This is all one eve version. So it makes sense.. And this file is actually generated by baseos mgr..